action.yml

```
@@ -84,14 +84,64 @@ branding:
   color: yellow
   icon: upload-cloud
 runs:
-  using: docker
-  image: Dockerfile
-  args:
-  - ${{ inputs.user }}
-  - ${{ inputs.password }}
-  - ${{ inputs.repository-url }}
-  - ${{ inputs.packages-dir }}
-  - ${{ inputs.verify-metadata }}
-  - ${{ inputs.skip-existing }}
-  - ${{ inputs.verbose }}
-  - ${{ inputs.print-hash }}
+  using: composite
+  steps:
```

I've been meaning to suggest that we fail early here in case the runner runtime is unsupported. Originally, GitHub Actions would error out for docker actions on invocation. But since the wrapper is now composite, it'd be nice to do this manually so that it won't waste CPU on platforms where it won't function anyway.

```suggestion
  steps:
  - name: Fail-fast in unsupported environments
    if: runner.os != 'Linux'
    run: |
      >&2 echo This action is only able to run under GNU/Linux environments
      exit 1
    shell: bash -eEuo pipefail {0}
```